### DISCUSSION TITLE
PR comments - CSS broken?

### DISCUSSION THREAD

**Discussion by billy1624** [community/community#33697] [Pull Requests]
The alignment of the emoji button seems off

<img width="959" alt="image" src="https://user-images.githubusercontent.com/30400950/191511043-a20a4683-f0b2-4416-8ac9-dce3590a28f1.png">

Link: https://github.com/SeaQL/sea-orm/pull/890

**Comment by Raykesh-Soneka** [comment]
Hi 
![image](https://github.com/user-attachments/assets/107bab19-268d-4b88-b1a2-371cdc466937)
it is working now ?
